Plugin/README.txt

```diff
@@ -22,7 +22,69 @@ NOTE: source tree requires submodules
     git submodule update --init --recursive
 
 ====================
-Building on Windows:
+Building the Dependencies:
+====================    
+    
+On Windows:
+
+Make sure to add this option when running cmake as 32 bit is often the default: -G"Visual Studio 14 2015 Win64"
+
+Building HDF5:
+
+1. follow instructions (Build Instructions 1-7) at https://portal.hdfgroup.org/display/support/Building+HDF5+with+CMake
+
+Note: make sure the binary is built in Release mode and is 64-bit
+
+2. This will generate a zip file. Unzip the file. The binaries will be located under lib/ and header files under include/
+
+3. copy the include/ directory into Plugin/external/HDF5
+
+4. copy the following library files to Plugin/external/HDF5/lib64: libhdf5.lib, libhdf5_hl.lib, libszip.lib
```

Comment:
Why install into an HDF5 directory instead of just into Plugin/external/lib64 and Plugin/external/bin and so on?

Reply:
it was how it was setup in the zip file I was using before. Thought I would leave the same layout. Otherwise I think I would need to modify the visual studio project as well.

Reply:
should I change it so that they should all be placed into Plugin/external/lib64 and Plugin/external/include?